berkeley-mobile/Resources/Campus/ResourceDetailViewController.swift

```diff
@@ -11,7 +11,7 @@ import Firebase
 
 fileprivate let kViewMargin: CGFloat = 16
 
-class ResourceDetailViewController: SearchDrawerViewController {
+class CampusResourceDetailViewController: SearchDrawerViewController {
```

Comment:
The file is still named `ResourceDetailViewController`, could you rename it to the new class name?